### PR TITLE
Read justified-formatted parameters txt

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Solving #53 by stripping whitespace from parameters.txt contents before parsing it. Also added testing for a negative float value, and renamed the `buffer` variable to `buffer_` to avoid overwriting the `buffer` type.

### CODE REVIEW

File: src/fmu/dataio/_utils.py [hunk 1]
```diff
@@ -191,6 +191,21 @@ def read_parameters_txt(pfile):
       LOG10_MULTREGT:MULT_THERYS_VOLON -3.21365
       LOG10_MULTREGT:MULT_VALYSAR_THERYS -3.2582
 
+    Or the structure::
```
Comment (jcrivenaes) [inline src/fmu/dataio/_utils.py:194]: Explain a bit more the background for this format (and perhaps use the word "format" not "structure"?)

File: src/fmu/dataio/_utils.py [hunk 2]
```diff
@@ -203,11 +218,18 @@ def read_parameters_txt(pfile):
         }
     """
 
+    logger.debug("Reading parameters.txt from {}".format(pfile))
+
     with open(pfile, "r") as stream:
-        buffer = stream.read().replace(" ", ":").splitlines()
+        buffer_ = stream.read().splitlines()
```
Comment (jcrivenaes) [inline src/fmu/dataio/_utils.py:224]: I don't understand this "buffer_" as the word "buffer" is not a reserved word in python(?)

**Review by jcrivenaes — APPROVED** [review]
It may be more elegant with a regexp, but OK for now

File: src/fmu/dataio/_utils.py (hunk 2 above)
Comment (perolavsvendsen) [reply src/fmu/dataio/_utils.py:224]: My IDE flagged it and I went on autopilot from there... Can revert it.

Comment (perolavsvendsen) [reply src/fmu/dataio/_utils.py:224]: Looks like Python 2.7 has it. Not present >3.5 as far as I can see. That's probably why it was flagged. 

https://docs.python.org/2.7/library/functions.html#buffer
